### DISCUSSION TITLE
bash: no job control in this shell

### DISCUSSION THREAD

**Discussion by davidecavestro** [Jip-Hop/jailmaker#258] [General]
Hi, I've a jail where I confined a docker installation, then used _dockge_ to manage some _compose.yml_.

This worked like a charm until I reinstalled TrueNas along with jailmaker and reused the existing jail from the data-pool.
From then on when I attach to the jail with a bash or any other shell, it complains for _Inappropriate ioctl for device/no job control in this shell_
This way:
```
admin@truenas:~$ jlmkr exec dockge /bin/bash
bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell
root@dockge:/# 
```
hence as soon as I issue a CTRL-C or other hotkeys I loose control of the terminal.

Everything else works fine, but I cannot find any evidence on the root cause for this, probably cause I've no experience with low level details of TTYs.
I know I can create a new jail, but I'd like to understand what's happening under the hood.

Any idea?

**Comment by Lockszmith-GH** [comment]
One thing you _can_ do is create a new jail and compare their rootfs - maybe there are some hard links there that went haywire.

**Reply by davidecavestro** [reply]
Unexpectedly, I obtained the same with the new jail, so the issue was not specific to that one, but probably to the spawn. I finally gave up and upgraded to EE.
